audit/projects/k8s-infra-prow-build-trusted/secrets/k8s-triage-robot-github-token/versions.json

```diff
@@ -0,0 +1,29 @@
+[
+  {
+    "createTime": "2021-07-23T03:18:11.506633Z",
+    "etag": "\"15c7c1da789dc9\"",
+    "name": "projects/180382678033/secrets/k8s-triage-robot-github-token/versions/3",
```

Comment:
As expected by https://github.com/kubernetes/k8s.io/pull/2389. Only one version is used.